test-core-2/upgrades/v11.7.0/binaries.json

```diff
@@ -0,0 +1,5 @@
+{
+  "binaries": {
+    "linux/amd64": "https://github.com/persistenceOne/persistenceCore/releases/download/v11.7.0/persistenceCore-v11.7.0-linux-amd64?checksum=sha256:ecbe82dc40bd15716d2a8cc67d3b388333436f382c7c9dc5aad362a1cc19f372"
```

Comment:
Link doesn't work, missing the `-fix` tag in the version

Reply:
It's already part of the chain state... this is a past upgrade